README.md

```diff
@@ -57,8 +57,57 @@ resources of this module:
 The [Project Factory module][project-factory-module] can be used to
 provision a project with the necessary APIs enabled.
 
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| host\_project | The network host project ID | string | `""` | no |
+| image\_family | Source image family for the Bastion. | string | `"centos-7"` | no |
+| image\_project | Project where the source image for the Bastion comes from | string | `"gce-uefi-images"` | no |
+| labels | Key-value map of labels to assign to the bastion host | map | `{}` | no |
+| machine\_type | Instance type for the Bastion host | string | `"n1-standard-1"` | no |
+| members | List of IAM resources to allow access to the bastion host | list | `[]` | no |
+| name | Name of the Bastion instance | string | `"bastion-vm"` | no |
+| network | Self link for the network on which the Bastion should live | string | n/a | yes |
+| project | The project ID to deploy to | string | n/a | yes |
+| region | The primary region where the bastion host will live | string | `"us-central1"` | no |
+| scopes | List of scopes to attach to the bastion host | list | `[ "cloud-platform" ]` | no |
+| service\_account\_roles | List of IAM roles to assign to the service account. | list | `[ "roles/logging.logWriter", "roles/monitoring.metricWriter", "roles/monitoring.viewer", "roles/compute.osLogin" ]` | no |
+| service\_account\_roles\_supplemental | An additional list of roles to assign to the bastion if desired | list | `[]` | no |
+| shielded\_vm |  | string | `"true"` | no |
+| startup\_script | Render a startup script with a template. | string | `""` | no |
+| subnet | Self link for the subnet on which the Bastion should live. Can be private when using IAP | string | n/a | yes |
+| zone | The primary zone where the bastion host will live | string | `"us-central1-a"` | no |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| hostname | Host name of the bastion |
+| ip\_address | Internal IP address of the bastion host |
+| self\_link | Self link of the bastion host |
+| service\_account | The email for the service account created for the bastion host |
+
+<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+
 ## Contributing
 
+### Pre-Commit
```

Comment:
While we *can* include pre-commit hooks I'd like us to follow the CFT standard which is to use the Docker commands to run these: https://github.com/terraform-google-modules/terraform-google-module-template/blob/master/terraform-google-%7B%7Bcookiecutter.module_name%7D%7D/CONTRIBUTING.md

Reply:
In fact, it looks like this is causing linters to fail. Could we follow the standard tools please?

Reply:
Sure, but we use the same pre-commit scripts in the docker command. I just had the wrong options, which is why it was failing.

Reply:
Got it. To be clear, I'm fine with us including pre-commit hooks but the contribution instructions should be based on the standard Docker workflow.

Reply:
Updated here: https://github.com/terraform-google-modules/terraform-google-bastion-host/pull/22